src/main/modules/session/index.test.ts

```diff
@@ -100,7 +100,7 @@ describe('session', () => {
       expect(mockCreateClient).toHaveBeenCalledWith({
         host: 'MOCK_REDIS_HOST',
         password: 'MOCK_REDIS_KEY',
-        port: 6380,
+        port: 6379,
```

Comment:
Please do not change this port as it corresponds to higher env not local. Suggested to create a new env variable to set this port (Not required as of now can be taken as clean-up process)